tests/MetricBundle/Fixtures/kernel_setup.yml

```diff
@@ -1,6 +1,5 @@
 parameters:
   secret: test
-  kernel.project_dir: '%kernel.root_dir%'
```

Comment:
This parameter doesn't exist since Symfony 5.0

Reply:
it doesn't, but tests are run for different versions of symfony, some of them are not aware of `%kernel.root_dir%`, which lead to invalid directory setup which pollutes workdir when tested locally

Reply:
But with this option some tests is broken in environment with symfony 5. This is failed result of the test on the branch with returned back "kernel.root_dir" parameter: https://github.com/lamoda/php-metrics/runs/2314113293?check_suite_focus=true

Reply:
Oh, I've got the problem. `kernel.root_dir` doesn't exist, not `kernel.project_dir`. Ok, I think we can keep this, I'll take a look into this problem later